src/Components/Header/Header.scss

```
@@ -0,0 +1,126 @@
+@import "../../Styles/common.scss";
+.header {
+  width: $main-width;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  height: 156px;
+  margin-bottom: 10px;
+  /* header Menu List */
+  .header__menu {
+    display: flex;
+    justify-content: flex-end;
+    padding: 15px 10px;
+    font-size: 12px;
+
+    li a {
+      padding: 0 10px 0 10px;
+      font-weight: 600;
+    }
+    li:nth-child(1) a {
+      color: $purple-text-color;
+    }
+    li:last-child a {
+      padding-right: 0;
+    }
+    li::after {
+      content: "";
+      float: right;
+      width: 1px;
+      height: 13px;
+      background-color: #d8d8d8;
+    }
+    li:last-child::after {
+      display: none;
+    }
+  }
+  /* header Logo */
+  .header__logo {
+    width: $main-width;
+    height: 63px;
+    margin-bottom: 3rem;
+
+    img:first-child {
+      position: absolute;
+      top: 1%;
+      width: 163px;
+      height: 22px;
+    }
+    img:last-child {
+      position: absolute;
+      width: 103px;
+      height: 79px;
+      top: 2%;
+      left: 50%;
+      transform: translateX(-50%);
+    }
+  }
+  /* gnb List */
+  .gnb {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    .gnb__menu__lists {
+      width: inherit;
+      display: flex;
+      justify-content: space-between;
+      .gnb__menu__list:first-child::before {
+        font-family: "Font Awesome 5 Free";
+        font-weight: 900;
+        font-size: 20px;
+        content: "\f0c9";
+        cursor: pointer;
+      }
+
+      .gnb__menu__list a {
+        padding: 0 2.5rem 0 2.5rem;
```

아래와 같이 간결하게 작성해주세요 ;) 
```suggestion
        padding: 0 2.5rem;
```